fusion-cli/build/babel-plugins/babel-plugin-i18n/test/index.test.js

```diff
@@ -53,3 +53,59 @@ export default function() {
     )
   ).toThrow('The translate component must have props.id be a string literal.');
 });
+
+test('babel-plugin-i18n - valid usage of useTranslations', () => {
```

Comment:
Purpose of tests in this file:
- static analysis finds dynamic and static strings
- static analysis is added to Set
- error is thrown with improper usage of `translate`